std/assembly/string.ts

```diff
@@ -380,6 +386,36 @@ export class String {
     );
     return out;
   }
+
+  repeat(count: i32 = 0): String {
+    assert(this !== null);
+    var length = this.length;
+
+    // Most browsers can't handle strings 1 << 28 chars or longer
+    if (count < 0 || length * count > (1 << 28)) {
+      throw new RangeError("Invalid count value");
+    }
+
+    if (count === 0 || !length) return EMPTY;
+    if (count === 1) return this;
+
+    var result = allocate(length * count);
```

Comment:
Isn't this missing the leading 32-bit length?

Reply:
Hmm. I use this [allocation method](https://github.com/MaxGraey/assemblyscript/blob/749ae3ecde02e2cd27025276478243bc819da014/std/assembly/string.ts#L7) which already include HEADER size during allocation if I right understand your comment =)

Reply:
Ah, yeah, makes sense then.